### PR TITLE
Leaf 3880 -disabled actions deceive 

### PR DESCRIPTION
**Pull request by jampaul3**

### CODE REVIEW

**Review by raphaelfretter — APPROVED** [review]

**Review by pete-nerantzinis — APPROVED** [review]

File: LEAF_Request_Portal/sources/System.php [hunk 1]
```diff
@@ -686,8 +686,16 @@ public function addAction()
...
+        $sql = 'INSERT INTO `actions` (`actionType`, `actionText`,
+                    `actionTextPasttense`, `actionIcon`, `actionAlignment`, `sort`, `fillDependency`)
+                VALUES (:actionType, :actionText, :actionTextPasttense, :actionIcon, :actionAlignment, :sort, :fillDependency)
+                ON DUPLICATE KEY UPDATE `actionText` = :actionText,
+                    `actionTextPasttense` = :actionTextPasttense,
+                    `actionIcon` = :actionIcon,
+                    `actionAlignment` = :actionAlignment, `sort` = :sort,
+                    `fillDependency` = :fillDependency, `deleted` = 0';
+
+        $this->db->prepared_query($sql, $vars);
```
Comment (aerinkayne) [inline LEAF_Request_Portal/sources/System.php:698]: I think we might want to check the deleted status and only update an existing one if it's disabled.  The action text (that the table is ordered by) and actionType aren't the same, so users might unintentionally overwrite an existing action if we just post it.

**Review by maxrichardjr — APPROVED** [review]

File: LEAF_Request_Portal/sources/System.php (hunk 1 above)
Comment (jampaul3) [reply LEAF_Request_Portal/sources/System.php]: Functionality has been added to check for this.

File: LEAF_Request_Portal/admin/templates/mod_workflow.tpl [hunk 2]
```diff
@@ -1210,62 +1211,104 @@
...
+        getUsedActionType(actionType, function (res) {
+            console.log(res);
+            let workflows = '';
+
+            for (let i in res.data) {
+                workflows += res.data[i].description + "<br />";
+            }
+
+            if (res.status.code == 2 && res.data.length) {
+                console.log('something to display');
```
Comment (Pelentan) [inline LEAF_Request_Portal/admin/templates/mod_workflow.tpl:1273]: Did you want to leave this in?  If so, perhaps something more descriptive?

**Review by Pelentan — CHANGES_REQUESTED** [review]

**Review by maxrichardjr — APPROVED** [review]

File: LEAF_Request_Portal/admin/templates/mod_workflow.tpl (hunk 2 above)
Comment (jampaul3) [reply LEAF_Request_Portal/admin/templates/mod_workflow.tpl]: Cleaned up.

**Review by Pelentan — APPROVED** [review]

**Review by maxrichardjr — APPROVED** [review]

**Review by shaneodd — APPROVED** [review]

**Review by raphaelfretter — APPROVED** [review]

**Review by AmandaSumner — APPROVED** [review]

**Pull request merged**